### DISCUSSION TITLE
GSoC: Reaching out to mentors of Project 14 (Add time-series anomaly detection support to Anomalib)

### DISCUSSION THREAD

**Discussion by Shakib-IO** [openvinotoolkit/openvino#23222] [Google Summer of Code]
Hi Dick Ameln, Samet Akcay, Ashwin Vaidya,
I hope you are doing well. I am doing my masters in computer science at Concordia University. I would like to discuss some details about this project and, if possible, receive some feedback about my application before submitting it.
Thanks for your help.

**Comment by adrianboguszewski** [comment]
@djdameln @samet-akcay @ashwinvaidya17 please look at it

**Comment by Shakib-IO** [comment]
Hello @djdameln @samet-akcay @ashwinvaidya17 
I'm eager to express my gratitude for the opportunity to connect with you. I'm interested in contributing to these two projects:

14. Incorporating time-series anomaly detection support into Anomalib.
16. Developing Synthetic Anomaly Generation for Threshold Selection and Model Evaluation within Anomalib.

Let's discuss and devise strategies to determine which project would be ideal for this year. Your feedback is highly valued, and I'm thrilled to have the chance to connect with you.

Thanks.

**Comment by Shakib-IO** [comment]
Hello @djdameln @samet-akcay @ashwinvaidya17 @adrianboguszewski 
I'm eager to express my gratitude for the opportunity to connect with you. I'm interested in contributing to this project:

14. Incorporating time-series anomaly detection support into Anomalib.

Let's discuss and devise strategies to determine which project would be ideal for this year. Your feedback is highly valued, and I'm thrilled to have the chance to connect with you.

Thanks.

**Comment by samet-akcay** [comment]
Hi @Shakib-IO, you could refer to this paper about what sort of models and datasets we are interested in this project.
https://arxiv.org/pdf/2302.12245.pdf

**Comment by Shakib-IO** [comment]
Hi @samet-akcay, @djdameln, @ashwinvaidya17
I have my first draft of the GSoC proposal. Could you please review this one and provide feedback? 
Let me know if anything needs to be added. 
Thanks. 
[Shakib_GSoC_2024.pdf](https://github.com/openvinotoolkit/openvino/files/14748391/Shakib_GSoC_2024.pdf)
